### ISSUE TITLE
Failed to set presence: No message sent

### ISSUE THREAD

**Issue by jub0t** [jewlexx/discord-presence#12]
**Describe the bug**
It prints the following instead of showing presence:

```shell
Failed to set presence: No message sent
```

**Code**
```rust
#![cfg_attr(
    all(not(debug_assertions), target_os = "windows"),
    windows_subsystem = "windows"
)]

use discord_presence::Client as DiscordRPC;
use std::env;

fn main() {
    let mut drpc = DiscordRPC::new(1002568032041832578);

    drpc.on_ready(|_ctx| {
        println!("READY!");
    });

    drpc.on_error(|ctx| {
        eprintln!("An error occured, {}", ctx.event);
    });

    drpc.start();

    if let Err(why) = drpc.set_activity(|a| {
        a.state("Running examples").assets(|ass| {
            ass.large_image("1024x1024")
                .large_text("Building a Discord Bot")
                .small_image("512x512")
                .small_text("With Hydrazine")
        })
    }) {
        println!("Failed to set presence: {}", why);
    }

    tauri::Builder::default()
        .run(tauri::generate_context!())
        .expect("Unknown Error Occurred While Running Application!");
}
```

- Windows 10

**Comment by jewlexx** [comment]
Hey sorry for the late response. I am away for the next couple of days and I can do my best to figure out the problem while I am. 

It appears that the RPC client is failing to interact with Discord for some reason or another. 

I'll have a look later today if I can and get back to you

**Comment by jewlexx** [comment]
@jareer12 Should be fixed in v0.5.6. If you could test it and get back to me would be greatly appreciated!

**Comment by jub0t** [comment]
It works now, thanks a lot 👍🏽 

![image](https://user-images.githubusercontent.com/57962439/182202525-384f38af-5833-4a1a-9157-ef3b137f5200.png)
![image](https://user-images.githubusercontent.com/57962439/182202557-141d94f4-65b1-4c92-a73f-2bf117b02ab2.png)
